migration-guide.adoc

```diff
@@ -330,6 +330,8 @@ Hibernate now reports many more kinds of `DiagnosticEvent` to JFR.
 was removed in favor of `org.hibernate.metamodel.MappingMetmodel` or `org.hibernate.metamodel.model.domain.JpaMetamodel`
 * Removed `AdditionalJaxbMappingProducer` in favor of `AdditionalMappingContributor`.
 * Removed `MetadataContributor` in favor of `AdditionalMappingContributor`
+* Removed incubating `setOrder()` from `SelectionQuery()` in favor of `SelectionSpecification.sort()`
+* `PostgreSQLIntervalSecondJdbcType` was moved to an internal package--use `@JdbcTypeCode(INTERVAL_SECOND)`
```

Comment:
Maybe list a few more, to allow web search / CTRL + F? I know _I_ would be more likely to find this through search than by reading the whole document.

```suggestion
* Various JDBC types were moved to an internal package -- use `@JdbcTypeCode` instead of `@JdbcType` to map your attributes to these types.
  For example use `@JdbcTypeCode(SqlTypes.INTERVAL_SECOND)` instead of `@JdbcType(PostgreSQLIntervalSecondJdbcType.class)`.
  Other moved types include `PostgreSQLEnumJdbcType`, `PostgreSQLOrdinalEnumJdbcType`, `PostgreSQLUUIDJdbcType`, `DB2StructJdbcType`, `H2DurationIntervalSecondJdbcType`, `H2JsonJdbcType`, `OracleBooleanJdbcType`, `OracleEnumJdbcType`, `OracleJsonJdbcType`, `OracleOrdinalEnumJdbcType` `OracleReflectionStructJdbcType`, `OracleXmlJdbcType`.
```